docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -470,6 +470,9 @@ following properties:
 * - `iceberg.rest-catalog.warehouse`
   - Warehouse identifier/location for the catalog (optional). Example:
     `s3://my_bucket/warehouse_location`
+* - `iceberg.rest-catalog.namespace`
```

Comment:
This config property will be used as "parent" parameter of REST call. `iceberg.rest-catalog.parent-namespace` might be better.